### ISSUE TITLE
the selection of optimal model

### ISSUE THREAD

**Issue by pray5959** [cchen-cc/SIFA#19]
Hi, thanks for your good work. What's the specific criterion for selecting the optimal model? The highest 'dice_fake_b'?

**Comment by cchen-cc** [comment]
Hi, it's selected based on the highest 'dice_b'.

**Comment by pray5959** [comment]
Is that the dice of target validation set? I saw in main.py that there only the dice of source validation set is summarized.

**Comment by cchen-cc** [comment]
It‘s the dice of target training set. As the target labels are not used during training, so we didn't split a validation set.
